Documentation/Contributing/ExperimentalFeatures.md

```diff
@@ -59,6 +59,13 @@ Adding a component menu:
 [AddComponentMenu("MRTK/Experimental/MyCommand")]
 ```
 
+### Documentation
+Follow these steps to add documentation for your experimental feature:
+
+1. Any documentation for an experimental feature should go in a `README.md` file in the experimental folder. For example, [`MixedRealityToolkit.SDK/Experimental/ScrollingObjectCollection/README.md`](../../Assets/MixedRealityToolkit.SDK/Experimental/ScrollingObjectCollection/README.md). 
+
+1. Add a link in the Experimental section at [`Documentation/toc.yml`](../toc.yml).
```

Comment:
I don't think there's actually a section called literally "Experimental" in the toc.yml. If we search for 'experimental' (ctrl+F) on that file there are two locations, one of which doesn't adhere to the rules (i.e. UnityAR) and another called "Experimental Features"

If we actually mean 'Experimental Features' we should be completely literal here (otherwise it can be hard to resolve ambiguities)

Reply:
we should also move existing experimental features into that section. (eg object collection)

Reply:
Have a look at https://github.com/microsoft/MixedRealityToolkit-Unity/pull/6943/files the toc.yml changes -- Yoon added scrolling object collection and pulse shader to a new experimental section.  But yes, this guidance should go in after #6943 is merged.